src/view/com/auth/create/StepHeader.tsx

```diff
@@ -2,13 +2,17 @@ import React from 'react'
 import {StyleSheet, View} from 'react-native'
 import {Text} from 'view/com/util/text/Text'
 import {usePalette} from 'lib/hooks/usePalette'
+import {useLingui} from '@lingui/react'
+import {Trans, msg} from '@lingui/macro'
 
 export function StepHeader({step, title}: {step: string; title: string}) {
   const pal = usePalette('default')
+  const {_} = useLingui()
+
   return (
     <View style={styles.container}>
       <Text type="lg" style={[pal.textLight]}>
-        {step === '3' ? 'Last step!' : <>Step {step} of 3</>}
+        {step === '3' ? _(msg`Last step!`) : <Trans>Step {step} of 3</Trans>}
```

Comment:
We can wrap both these strings in the `<Trans>` macro

Reply:
Instead of using `_(msg``)`

Reply:
Thanks, I fixed it.